doc/cheatsheet.mld

```diff
@@ -0,0 +1,225 @@
+{0 Cheatsheet}
+
+Quick reference for the odoc language!
+
+{table
+  {tr
+    {th }
+    {th [odoc] syntax}
+    {th Render as}}
+  {tr
+    {th Paragraphs}
+    {td {[
```

Comment:
The code blocks are highlighted in a broken way. Can you add the language annotation ? `{@text[ .. ]}` works well. `{@odoc[ .. ]}` is not recognized by the highlighter and renders like verbatim.

Reply:
Good point, thanks!